CHANGELOG.md

```diff
@@ -89,6 +95,7 @@
 - Differentiate `quicksetup` profile settings based on project folder [[#1901]](https://github.com/aiidateam/aiida_core/pull/1901)
 - Ensure `WorkChain` does not exit unless stepper returns non-zero value [[#1945]](https://github.com/aiidateam/aiida_core/pull/1945)
 - Fix variable `virtual_memory_kb` in direct scheduler. [[#2050]](https://github.com/aiidateam/aiida_core/pull/2050)
+- Fixed a bug when using bash login shells in transports that write spurious output [[#2132]](https://github.com/aiidateam/aiida_core/pull/2132)
```

Comment:
Only comment: this wasn't a real bug fix, mainly better documentation and an additional test in `verdi computer test`

Reply:
Shall I move it to documentation section?

Reply:
Ok